### DISCUSSION TITLE
When convert dxf to svg ,how to hide the text which is originally in dxf ,when generate svg? how to set the line to black?

### DISCUSSION THREAD

**Discussion by srillia** [mozman/ezdxf#775] [Q&A]
```
from ezdxf import recover
from ezdxf.addons.drawing import matplotlib
from ezdxf.addons.drawing.config import Configuration

# Exception handling left out for compactness:
doc, auditor = recover.readfile('nzk-1.dxf')

if not auditor.has_errors:
    config = Configuration.defaults()
    matplotlib.qsave(
        doc.modelspace(), 
        'nzk-1.svg',
        config=config.with_changes(pdmode=1),  # do not plot any point geometry
    )
```
![EGy2DeTYIK](https://user-images.githubusercontent.com/25891673/201017040-370e7e92-b495-431c-9c26-b7cecd7f01be.jpg)
The red box contains text, I don't need them in svg,but the text originally in dxf. Is there a way to set some param to avoid this.
![717OOd34q6](https://user-images.githubusercontent.com/25891673/201017673-2e74689d-d4bb-496b-9bd7-5c652605f956.jpg)
The red box conains white line. What I need is transprent background ,the black line. I alreay know how to set transprent, but I don't know how to set 'black' line.

**Comment by srillia** [comment]
the files in #765.

**Comment by mozman** [comment]
For a transparent background read the docs about the `bg` argument of the [qsave()](https://ezdxf.mozman.at/docs/addons/drawing.html#ezdxf.addons.drawing.matplotlib.qsave) function.

There is no specific function which solves exact your problem. But as programmer you have always multiple ways to achieve your goals if you know the topic and the tools you have.

1. if all text is on the same layer, switch off the layer
2. delete the text entities you don't wanna plot, but don't save the altered DXF file
3. use the `filter_func` argument of the `qsave()` function to exclude certain entities from plotting
